CHANGELOG.md

```diff
@@ -79,7 +90,7 @@ Release built: 23.10.2024
 - New configuration options `DataAggregator__Storage__StoreTransactionReceiptEvents`, and `DataAggregator__Storage__StoreReceiptStateUpdates` for the data aggregator to configure if a transaction's receipt events and receipt state updates should be stored in the database. It is meant to be used by gateway runners who want to reduce their database size. Keep in mind that when disabled, the corresponding properties will be missing on a response from both the `/stream/transactions` and the `/transaction/committed-details` endpoints. You can save significant space by using `StoryOnlyForUserTransactionsAndEpochChanges` and only excluding round change transactions, which aren't typically read from the `/stream/transactions` endpoint.
   - Possible values:
     - `StoreForAllTransactions` (default) - will store data for all transactions.
-    - `StoryOnlyForUserTransactionsAndEpochChanges` - will store data for user transactions and transactions that resulted in epoch change.
+    - `StoreOnlyForUserTransactionsAndEpochChanges` - will store data for user transactions and transactions that resulted in epoch change.
```

Comment:
Well... to be accurate, this should probably stay as `StoryOnlyForUserTransactionsAndEpochChanges` in the Changelog, with a highlight that the typo is fixed in 1.9.2 :D 

Indeed, I've already added a comment below which should be amended to say 1.9.2 instead of "future version"

Reply:
you're right, if someone would use 1.8.x he has to use value with typo :D reverted.